extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -404,9 +404,14 @@ function _jumpToMeasurement(
     displaySetInstanceUID
   );
 
-  const imageIdIndex = referencedDisplaySet.images.findIndex(
```

Comment:
Revert to the const imageIdIndex, but iterate over the imageIds instead of the images, and use the MetadataProvider to get the sop uid and frame# for the given image id.

Reply:
Fixed it in https://github.com/OHIF/Viewers/pull/2973/commits/75b4cfcef27b972745ba2e3ea3f9546d5f8cf27d